### PR TITLE
Develop blog

### PR DESCRIPTION
**Pull request by activefog**

### CODE REVIEW

**Comment by dennisreimann** [comment]
Awesome, this is definitely going in the right direction -- good job!

File: blog/_posts/bitcoin-pos.md
Comment (Zaxounette) [inline blog/_posts/bitcoin-pos.md:3]:
```suggestion
author: Pavlenex
```

File: blog/_posts/btcpay-misconceptions.md
Comment (Zaxounette) [inline blog/_posts/btcpay-misconceptions.md:3]:
```suggestion
author: Pavlenex
```

File: blog/_posts/mike-olthoff-coincards.md
Comment (Zaxounette) [inline blog/_posts/mike-olthoff-coincards.md:3]:
```suggestion
author: Pavlenex
```

File: blog/_posts/payment-requests.md
Comment (Zaxounette) [inline blog/_posts/payment-requests.md:3]:
```suggestion
author: Pavlenex
```

File: blog/_posts/ride-the-lightning-btcpay.md
Comment (Zaxounette) [inline blog/_posts/ride-the-lightning-btcpay.md:3]:
```suggestion
author: Pavlenex
```

Comment (activefog) [reply blog/_posts/ride-the-lightning-btcpay.md]: Thanks for pointing out these typos, @Zaxounette! I think we'll keep it lowercase to be consistent with pavlenex's preferred pseudonym styling, but I've now corrected the a/e typo in a subsequent commit.

File: blog/.vuepress/components/Categories.vue
Comment (dennisreimann) [inline blog/.vuepress/components/Categories.vue:8]: fyi: Vue has the concept of [filters](https://vuejs.org/v2/guide/filters.html) exactly for this.

Comment (activefog) [reply blog/.vuepress/components/Categories.vue]: Thanks Dennis -- I addressed this in the latest commits

Comment (dennisreimann) [reply blog/.vuepress/components/Categories.vue]: Great, looks good! 💪

**Comment by activefog** [comment]
@dennisreimann -- Something seems wonky with the linkcheck -- it's showing all github profile links as broken, as well as links to Kraken in the sponsors footer. Any ideas?

**Comment by Zaxounette** [comment]
Regarding the linkcheck.
* All the HTTP_429 can be ignored I think and added as exceptions, they work when I access them, they just get blocked by automation.
* All the kraken HTTP_503 will probably be added as exceptions by Dennis, like on the docs repo.
* Some of the 404 work, some are broken (the broken ones are 1 year+ old)

Other than that, there are a few links that are probably broken in your code:
```
├─BROKEN─ http://araspitzu/ (BLC_UNKNOWN)
├─BROKEN─ http://bolatovumar/ (HTTP_undefined)
```

**Comment by dennisreimann** [comment]
Yes, as Zax pointed out, the ones that aren't really broken should be excluded from the check, see the [exclude list in package.json](https://github.com/btcpayserver/blog/pull/5/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R19) for reference.

**Comment by dennisreimann** [comment]
Now the linkcheck works again :)

**Comment by dennisreimann** [comment]
This is so cool and looks great already! Very good job, @activefog 

Remaining TODOs I noticed:

- Search flyout in dark mode needs color fixes (similar to the pagination)
- The "Last Updated:" color at the end of an article doesn't fit
- The links in the subline "Posted in CATEGORY by AUTHOR on DATE" could use a more subtle color. 

Again, really cool -- looking forward to launching this!

Once the site is live we can apply for Algolia DocSearch, which might improve the search. Not sure if needed though. /cc @pavlenex

**Comment by activefog** [comment]
Thanks all -- I didn't realize there was a link-check exclusions list

Agreed @d11n -- latest commit updates the styles you mentioned. Good catch on the search flyout -- I hadn't actually tried searching yet.

Another bug I'm noticing is that if you go into page 2, etc., of the posts index, you see "Page 2 | | BTCPay ..." (i.e., two pipes instead of one) as the page title in the browser tab. I was able to fix this *locally* by deleting the pipe on line 56 of the handleOptions.js file in the vuepress-plugin-blog module, but short of forking that module and adding the fork to package.json, I'm not sure how to get this to work in the build process.  Any ideas on how to monkey patch this?  Or perhaps something else in the code that I'm missing?

**Comment by dennisreimann** [comment]
@activefog I fixed that  "Page 2 | | BTCPay" issue in b797f0f. 

Nevertheless there is one more place the fix does not work: The index page of a category, which does not get the title via the [`getPaginationPageTitle` function](https://vuepress-plugin-blog.ulivz.com/pagination/#getpaginationpagetitle). I'd suggest we create a PR to solve this -- there is a similar request already vuepress/vuepress-plugin-blog#90

**Review by dennisreimann — APPROVED** [review]
This is really awesome already. Reviewed it and it is good to go imho. We can improve minor stuff later, but I think this is ready for prime time now.

Very solid work, @activefog -- thanks for all the effort!

**Review by pavlenex — APPROVED** [review]
LGTM, let us know when you guys are ready to do a DNS migration.

Awesome work the blog looks really dope 🔥

**Review by Zaxounette — APPROVED** [review]

**Comment by dennisreimann** [comment]
@activefog If you think this is ready I'll merge. Preview will then be here: https://btcpayserver.github.io/blog/

Once this is merged we can do the DNS switch and point the A record to the four GH Pages IPs as for the website and directory.

**Comment by Kukks** [comment]
Can we have the sponsors how on every page?

**Comment by Zaxounette** [comment]
> Can we have the sponsors show on every page?

I second this.
Currently they are missing in the main page and the different category pages.

**Comment by activefog** [comment]
Thanks all -- I added sponsors to all pages.

In general, I think this all looks good, and I agree that it is ready to go.

One additional minor thing I see is that category page titles are not getting capitalized (in browser tabs). Frustratingly, I don't see a way to config that through vuepress-plugin-blog, which is creating the pages automatically (https://vuepress-plugin-blog.ulivz.com/config/ only shows a way to update the directory title, not the individual category pages, which I would expect to see as an "itemTitle" option).  I'm still learning Vue, and perhaps there is another trick to get this done that I am missing?  Either way, I don't think this needs to hold back the cut-over from the WordPress site which currently has worse bugs anyway :)

**Comment by Zaxounette** [comment]
Agreed, this annoying issue doesn't have to hold out the release of the new blog in it's current state.

Very nicely done, and quickly, thanks for that! <3

**Comment by dennisreimann** [comment]
@activefog yes, that's what I meant above, I guess we need to send in a fix as PR in the blog plugin. 

Let's get this out there first! Merging …

**Pull request merged**
